### DISCUSSION TITLE
Issue... again. sorry.

### DISCUSSION THREAD

**Discussion by chessbored** [FabricMC/fabric-api#2184] [Player support] [answered]
I have yet another issue... im sorry for not knowing anything. and ive tried previous methods that ive been given for this exit code. its the same one as two other times. im fully updated, ive redownloaded mc, restarted my pc, no overlays are on and ive deleted razer synapse. whats wrong now. 
[LOG](https://pastebin.com/CqRGZXxC)

**Comment by warjort** [comment]
The only thing that says error is
````
[02:03:30] [Worker-Main-9/ERROR]: Using missing texture, unable to load patchouli:textures/items/book_blue.png : java.io.FileNotFoundException: patchouli:textures/items/book_blue.png
````
but from what I can see you don't have patchouli in your mods?

You also still have a few warnings about minecraft assets which suggests your resource packs still have a problem somewhere.
Look at the WARNs at the end of your log. These are just warnings, but try using removing your custom resource packs to see which is causing the problem(s).

**Comment by warjort** [comment]
That error code -805306369 is ultimately a java issue, but it seems mainly related to virus protection interference when its not a memory problem?
You might get more insight if you ask whoever supplies your java or virus protection software?

**Reply by chessbored** [reply]
java supplies my java and i dont have any protection software :c

**Comment by chessbored** [comment]
ok so apparently it keeps saying im missing the texture reference for the air filter, but its there? i dont understand..

**Reply by warjort** [reply]
````
[02:03:29] [Worker-Main-6/WARN]: Unable to resolve texture reference: #missing in enva:block/air_filter
````
That's a bug in the "enva" mod. But its only a WARNing, it wont be the cause of your problem.

**Comment by warjort** [comment]
Is this relevant to your problem?
https://bugs.mojang.com/browse/MCL-20158

**Reply by chessbored** [reply]
no, sadly. im pretty sure it isnt

**Comment by chessbored** [comment]
i removed Sodium from the modlist and it started working fine?? but isnt sodium supposed to help?

**Reply by chessbored** [reply, marked as answer]
i have read up and it says if any of the mods i choose to use use Fabric's rendering api, it could crasg with sodium. i believe this is the issue.
